src/utils/index.js

```diff
@@ -52,3 +52,10 @@ export function unprefixKeys(obj, prefixToRemove, lowercaseFirst = true) {
     return prev;
   }, {});
 }
+
+/**
+ * Returns a className string containing all non-empty classes from the given list of candidates.
+ */
+export function joinClasses(...candidates) {
```

Comment:
i'd actually much rather use `classnames` - initially introduced it in https://github.com/Clever/components/pull/36#discussion-diff-73739880 but reverted it due to concerns about bloat. `joinClasses` is a compromise, but down to switch to `classnames` if there are no objections